hawk/app/assets/stylesheets/shared/_foot.scss

```diff
@@ -2,19 +2,26 @@
 // See COPYING for license.
 
 footer {
-  background-color: #fff;
-  padding: 20px 10px;
-  border-top: 1px solid #999;
+  display: none;
+
+  background-color: #f9f9f9;
+  padding: 15px 10px;
+  border-top: 1px solid #ccc;
```

Comment:
Color literals like `#ccc` should only be used in variable declarations; they should be referred to via variable everywhere else.